README.md

```diff
@@ -1,2 +1,36 @@
 [![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/cSGmFTKd)
-# INF 6420 - Intro to Web Development
+# Florence F. Cintron's Website Project
+A Portfolio showcasing my work
+
+## wireframes 
+
+Here are a few wireframes to show how the layout of the website might possibly look like. 
+
+![wireframes for landing page](wireframes/wireframe1.jpg) 
```

Comment:
Remove the space in your image filename. `wireframe1.jpg` instead of `wireframe 1.jpg` to match this code.